src/resources/items.py

```diff
@@ -0,0 +1,22 @@
+from .base import AbstractDungeonEntity
+from .character import Character
+
+
+class AbstractDungeonItem(AbstractDungeonEntity):
```

Comment:
Is this class an abstract class as well?

Reply:
Renamed to AbstractColorChangingItem, but yes it is for color changing items/entities